Automattic/wp-calypso
# add the campaign objective in the campaign details page

Related to #

## Proposed Changes

we want to add the campaign objective in the campaign details page

those are the 4 different options

<img width="385" alt="Screenshot 2024-09-20 at 13 07 28" src="https://github.com/user-attachments/assets/28c14a01-299b-42a3-8f58-d79ede102797">
<img width="400" alt="Screenshot 2024-09-20 at 13 07 38" src="https://github.com/user-attachments/assets/b057c70e-6500-4776-882f-b2fc0ff3303b">
<img width="438" alt="Screenshot 2024-09-20 at 13 07 47" src="https://github.com/user-attachments/assets/f1fba5af-edca-4bcf-afd1-fb5770a0ac2d">
<img width="410" alt="Screenshot 2024-09-20 at 13 07 56" src="https://github.com/user-attachments/assets/53c0a849-bf44-4c28-9c4d-04f2e6fdfaee">

the data is now fetched from the same endpoint that fetches the campaign data

## Testing Instructions

checkout this branch (or go to the preview link), create 4 campaigns with the different objectives (you don't need to online them .. canceling them is also fine)

go to each of the campaigns details page and check the result


*

## Pre-merge Checklist
